packages/javascript/bh-shared-ui/src/components/GroupManagementContent/GroupManagementContent.tsx

```diff
@@ -74,6 +72,24 @@ const GroupManagementContent: FC<{
 
     const selectedAssetGroup = listAssetGroups.data?.find((group) => group.id === selectedAssetGroupId) || null;
 
+    const { data: memberCounts } = useQuery({
+        queryKey: [
+            'getAssetGroupMembersCount',
+            filterParams.environment_id,
+            filterParams.environment_kind,
+            selectedAssetGroup,
+        ],
+        enabled: !!selectedAssetGroupId,
+        queryFn: ({ signal }) =>
+            apiClient
+                .getAssetGroupMembersCount(
+                    selectedAssetGroupId?.toString() ?? '', // TODO: this sucks, but dependent queries are better in react-query v4 and we're on v3
```

Comment:
Will this log net request errors when selectedAssetGroupId is undefined?  Do we care?

Reply:
This query should only run when the selectedAssetGroup is not falsey because we have it enabled depending on `!!selectedAssetGroup`. So in theory selectedAssetGroup shouldnt be undefined when this network request happens.  Ill update this comment to better describe whats going on here!

There way to do this that is [cleaner and more explicit in the v5](https://tanstack.com/query/v5/docs/framework/react/guides/dependent-queries#usequeries-dependent-query) version of react-query, but this enabled field is whats available to us in v4!

Thanks!